### ISSUE TITLE
Treasure Table Request for Improvement

### ISSUE THREAD

**Issue by BobHood** [gizmomathboy/hyperborea_apps#3]
I'm hoping that there is a simple way to create a treasure Horde. I might just not be reading the instructions correctly, but I'd love it if I could just punch in a bunch of letters and have it compile all the results into a single table (adding the like money amounts as one). 

Ex, The party encounters 10 Skeletons (2 large), 3 zombies, and a Litch controlling them. They kill them all and the treasure would be 
14xQ, 2x(J,K,L,M), U
Even if I spelled it out that would be great. (ex instead of 14xQ = Q,Q,Q,Q,Q,Q,Q,Q,Q,Q,Q,Q,Q,Q...)
Of course, I then go to the page and find the way to do it, But Still in the request to have it Sum all the treasure would nice, even if it was Is a horde checkbox to sum it all together.

**Comment by gizmomathboy** [comment]
Yep, you can generate a horde, the syntax from your example would be:

Qx14 Jx2 Kx2 Lx2 Mx2 U

**Comment by gizmomathboy** [comment]
those can be lower case as well:

qx14 etc

...I tried xx2 but that kinda didn't go well :-)

but LETTERxNUMBER should handle repeated stuff

**Comment by BobHood** [comment]
Too bad there isn't a "Sum" button that puts all the treasure as a single table at the end...

My Players are beginning to hate me when I say 2 gp 4 gp 9 gp 2 gp x x x x x...

**Comment by gizmomathboy** [comment]
I'll look into it. The tricky part is jewelry and maybe gems (I can't recall at the moment if they go multi-line like jewelry).

**Comment by gizmomathboy** [comment]
and added.

Other summaries might not be...doable.
